### PR TITLE
ui: new header, footer, homepage

### PR DESCRIPTION
**Pull request by harunurhan**

### CODE REVIEW

**Comment by harunurhan** [comment]
it's WIP at the moment.

File: ui/src/common/constants.js [hunk 1]
```diff
@@ -21,8 +21,22 @@ export const LITERATURE_PID_TYPE = 'literature';
...
 export const CONFERENCES_PID_TYPE = 'conferences';
 
+export const INVENIO_URL = '//invenio-software.org';
+export const HOLDINGPEN_URL = '/holdingpen';
+export const AUTHORLIST_TOOL_URL = '/toos/authorlist';
+export const INSPIRE_TWITTER_ACCOUNT = 'https://twitter.com/inspirehep';
 export const SURVEY_LINK = 'https://forms.gle/CMgi3UsbpMuXxAfr6';
 export const FEEDBACK_EMAIL = 'feedback@inspirehep.net';
+export const BLOG_URL = 'https://blog.inspirehep.net';
+export const HELP_BLOG_URL = 'https://labs.inspirehep.net/help';
```
Comment (drjova) [inline ui/src/common/constants.js]: do we want this hardcoded? we'll change soon to ``inspirehep.net``

File: ui/src/common/layouts/Header/Header.jsx [hunk 2]
```diff
@@ -10,53 +10,50 @@ import BetaInfoBanner from './BetaInfoBanner';
...
-          </Col>
-        </Row>
-      </Layout.Header>
+    <div className="__Header__">
+      <div ref={stickyContainerRef} className="sticky">
+        <InterventionBanner />
+        {isBetaPage && (
+          <>
+            <BetaInfoBanner />
+            <BetaRibbon />
```
Comment (drjova) [inline ui/src/common/layouts/Header/Header.jsx]: Can we just get rid of them from now?

File: ui/src/home/index.jsx [hunk 3]
```diff
@@ -1,48 +1,52 @@
...
 
 import SearchBoxContainer from '../common/containers/SearchBoxContainer';
 import HowToSearch from './components/HowToSearch';
 import './index.scss';
 import DocumentHead from '../common/components/DocumentHead';
+import HomePageSection from './components/HomePageSection';
+import HowToSubmit from './components/HowToSubmit';
+import CollaborationLogos from './components/CollaborationLogos';
 
+const INSPIRE_DESCRIPTION =
```
Comment (drjova) [inline ui/src/home/index.jsx]: can we read them from the config instead and default to this?

**Comment by drjova** [comment]
Screenshot 2020-03-18 at 13 31 14

on mobile two rows might be better

**Comment by harunurhan** [comment]
@drjova  we can put `gutter` to ensure enough space between them and leave the rest adapt the screen, it will go the next row if there isn't enough space.

**Comment by drjova** [comment]
@harunurhan makes sense

**Comment by harunurhan** [comment]
@drjova  didn't work quite well 😄

File: ui/src/common/layouts/CollectionsMenu/index.js [hunk 4]
```diff
@@ -0,0 +1,3 @@
+import CollectionsMenu from './CollectionsMenu';
+
+export default CollectionsMenu;
```
Comment (drjova) [inline ui/src/common/layouts/CollectionsMenu/index.js]: I don't like this end of the line but is everywhere i guess
 :P

File: ui/src/common/layouts/Header/Header.jsx (hunk 2 above)
Comment (drjova) [reply ui/src/common/layouts/Header/Header.jsx]: https://its.cern.ch/jira/browse/INSPIR-3288

File: ui/src/common/layouts/CollectionsMenu/index.js (hunk 4 above)
Comment (harunurhan) [reply ui/src/common/layouts/CollectionsMenu/index.js]: well it's `js`, you have to export to be able import 😄

Comment (drjova) [reply ui/src/common/layouts/CollectionsMenu/index.js]: Screenshot 2020-03-18 at 14 38 40

Comment (drjova) [reply ui/src/common/layouts/CollectionsMenu/index.js]: I didn't mean the export :P

File: ui/src/common/constants.js (hunk 1 above)
Comment (harunurhan) [reply ui/src/common/constants.js]: well this is one of the step to update it, I moved it to a single place.

File: ui/src/home/index.jsx (hunk 3 above)
Comment (harunurhan) [reply ui/src/home/index.jsx]: maybe when we plan to configure it, we can do that.

File: ui/src/common/layouts/Header/Header.jsx (hunk 2 above)
Comment (harunurhan) [reply ui/src/common/layouts/Header/Header.jsx]: I think it's better to do it separately, it's not just removing the ribbon, but also modifying the banner from `beta` to `new release`, with different info in it.

**Pull request merged**